app/views/user_mailer/form_feedback.html.erb

```diff
@@ -0,0 +1,17 @@
+<p>
+    You recently archived the form, <%= @form.title %> in Touchpoints.
```

Comment:
I feel like there should be a comma after `@form.title` too.

Reply:
incorporated